src/main/java/com/solvd/util/XmlParser.java

```diff
@@ -0,0 +1,38 @@
+package com.solvd.util;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
+
+public class XmlParser {
+    public static void Parser(String xmlFilePath) {
```

Comment:
```suggestion
    public static void parse(String xmlFilePath) {
```
method names must start with lowercase letter